src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateGoalTest.kt

```diff
@@ -92,8 +97,12 @@ class CreateGoalTest : IntegrationTestBase() {
     assertThat(actionPlan).isForPrisonNumber(prisonNumber)
     assertThat(actionPlan).hasNumberOfGoals(1)
     val goal = actionPlan!!.goals!![0]
-    assertThat(goal).hasTitle(createRequest.title)
-    assertThat(goal).hasNumberOfSteps(createRequest.steps.size)
+    assertThat(goal).hasTitle(createGoalRequest.title)
```

Comment:
You are not a fan of chaining the assertions are you? 🤣

Reply:
Ah yes, I did say I'd change that in this PR! I appreciate chaining can read nicely 👍